lms/resources/_js_config/__init__.py

```diff
@@ -238,6 +239,7 @@ def enable_file_picker_mode(self, form_action, form_fields):
                 "filePicker": {
                     "formAction": form_action,
                     "formFields": form_fields,
+                    "promptForTitle": prompt_for_title,
```

Comment:
Adding this here at the top level.

This is our own FE/BE interface, easy to change later.

For now the simplest approach works.